app/src/main/java/app/nexd/android/ui/helper/detail/HelperDetailViewModel.kt

```diff
@@ -84,4 +105,15 @@ class HelperDetailViewModel(private val api: Api) : ViewModel() {
         super.onCleared()
         compositeDisposable.clear()
     }
+
+    object ButtonTextStyleBinding {
```

Comment:
unused, please remove and optimize imports. In general viewmodel should not know anything about UI (e.g. Buttons, Textviews etc.). The concept of viewmodel forces it to prrovide/manage the data that should be displayed without knowing HOW it will be displayed (e.g. in a button, an imageview or even just a text based terminal application).

Reply:
The object will be used at compile time with the creation of the data binding classes.

This object is being used for changing the font style (normal <> bold) via databinding depending on, if the helper has accepted it or if it is an open request (accepted: bold, not-accepted: normal).

Reply:
Ah i see, then my android studio just did not get the reference in bindings right. 

I think we can make it even simplier. We can use an extension for TextView:

``` kotlin
fun TextView.setTextStyle(style: Int) {
    this.setTypeface(typeface, style)
}
```

Put this extension in `src/main/java/app/nexd/android/ui/utils/extensions/TextView.kt`

This way you can call `android:textStyle=@{viewmodel.textStyle}` wicht can be an integer of TypeFace.BOLD etc.

Reply:
i moved the binding to `src/main/java/app/nexd/android/ui/utils/DataBindingAdapter` thinking it might fit better there.

Currently, the textStyle of the the text of the button is set conditionally `android:textStyle='@{!viewModel.requestIsAccepted? "normal" : "bold"}'`, so depending on, if it's the user's request, it will be normal else it will be bold. Unfortunately, I dont know how to use TypeFace constant values (TypeFace.NORMAL, etc.) in the layout file or it doesn't work that way.

Additionally, I added the other styles (italic, bold_italic) to possible text styles for button (just in case it might be needed somewhere else).

Reply:
as discussed, the programming if-logic has been moved from the layout to the underlying fragment and the corresponding DataBindingAdapter has been removed.

For the sake ok github CI gradle build failure the class name of DefaultSnackbar has been renamed to DefaultSnackBar (capital 'B' in bar).